apps/website/src/components/show-and-tell/ShowAndTellEntryForm.tsx

```diff
@@ -71,9 +73,12 @@ type ShowAndTellEntryFormProps = {
 function ImageAttachment({
   entry,
   fileReference,
+  callback,
```

Comment:
Should this be called `onClick`, and then exclude `onClick` from `ComponentProps` below?

Reply:
Yes, I think that makes more sense!